lib/vscode/product.json

```diff
@@ -28,7 +28,8 @@
 		"ms-vscode.remotehub",
 		"ms-vscode.remotehub-insiders",
 		"GitHub.remotehub",
-		"GitHub.remotehub-insiders"
+		"GitHub.remotehub-insiders",
```

Comment:
@kylecarbs This will need to be adjusted in `https://github.com/cdr/vscode/blob/code-server/product.json` and merged into the `code-server` branch.

Reply:
Gotcha. Doing it now!

Reply:
Actually I'll tackle this after the all-hands. Gotta prep!

Feel free to tackle if you have time!